packages/components/atoms/f-form-field/src/components/FormDropdown.vue

```diff
@@ -2,16 +2,20 @@
     <div
         :data-test-id="testId.container">
         <caret-icon
-            :class="$style['c-formDropdown-icon']"
+            :class="[
+                $style['c-formDropdown-icon'],
+                $style[`c-formDropdown-iconIndent--${fieldSize}`]
```

Comment:
You could do the same as below for these size classes, but you'd need to move the `caret-icon` element to be after the select element (so not sure if that's possible, or if it'd break the styling already setup for that)